### ISSUE TITLE
Constructing a building throws error

### ISSUE THREAD

**Issue by RobinJayaswal** [dartmouth-cs98/18w-si32#189]
```
 File "/Users/robinjayaswal/Development/98/18w-si32/worker/game/Rules.py", line 62, in update_build_command
    self.players[move.playerId].decrement_resource(BUILDING_COST)
AttributeError: 'Player' object has no attribute 'decrement_resource'
```

**Comment by gabeboning** [comment]
fixed in 40dfa666028d2dca4f91334321232dbd0263702b

**Comment by gabeboning** [comment]
![image](https://user-images.githubusercontent.com/1010522/37014811-2598089a-20d1-11e8-888f-b85f77190e4a.png)
sniped
